src/SubtitleEdit.csproj

```diff
@@ -65,9 +65,9 @@
     <Reference Include="System.Web.Services" />
     <Reference Include="System.Windows.Forms" />
     <Reference Include="System.Xml" />
-    <Reference Include="zlib.net">
-      <HintPath>packages\zlib.net.1.0.4.0\lib\zlib.net.dll</HintPath>
-      <Private>False</Private>
+    <Reference Include="zlibnet, Version=1.3.0.0, Culture=neutral, processorArchitecture=MSIL">
+      <SpecificVersion>False</SpecificVersion>
+      <HintPath>packages\zlibnet.dll</HintPath>

```

Comment:
NuGet downloads at compile time. Try it: remove src\packages\zlib.net.1.0.4.0, and run build.bat.
One of its messages will be "Installing 'zlib.net 1.0.4.0'.". That's why these packages are not controlled by git.